modos/api.py

```diff
@@ -451,44 +451,40 @@ def from_file(
         htsget_endpoint: Optional[str] = None,
     ) -> MODO:
         """build a modo from a yaml or json file"""
-        instances = parse_multiple_instances(Path(path))
-        # check for unique ids and fail early
-        ids = [inst.id for inst in instances]
-        if len(ids) > len(set(ids)):
```

Comment:
**question:** is this check not needed anymore? Do I understand correctly that in the case of multiple elements with the same id in the file, we will silently override previous ones?